README.md

```diff
@@ -6,11 +6,63 @@ OpenAirplay is a python implementation of the Apple Airplay client, intended to
 
 I have always been annoyed about how iOS, Mac OSX, and all of Apples proprietary hardware/software has inhibited so many of us technology savvy people.  
 
+## Getting Started
+
+1. Clone the repository.  
+```
+$ git clone https://github.com/openairplay/openairplay.git
+```
+
+2. Initialize and pull sub-modules.  
+```
+$ git submodule update --init --recursive
+```
+
+3. Create virtual environment directory.  
+__Note:__ you can also create the virtual python environment directly within the directory containing the openairplay project if you would like.  
+```
+$ cd ~/ && mkdir python_envs && cd python_envs
```

Comment:
I would suggest just going with the virtualenv inside the project folder for development, I normally use `./venv`, and loading that could be added to run.sh to avoid manually sourcing each time / running from outside the terminal.

Reply:
Updated with requested changes.